container/dangerzone.py

```diff
@@ -24,83 +26,92 @@
 
 import magic
 
-# timeout in seconds for any single subprocess
-DEFAULT_TIMEOUT: float = 120
+# The size threshold (in MiB) over which we assume that it takes more than one
+# "operation" on a file.
+SIZE_THRESHOLD: float = 1.0
```

Comment:
I had to read this a couple of times to understand what it meant. I think it would be easier to understand if we made it `timeout_per_mb`:

```suggestion
# The size threshold (in MiB) over which we assume that it takes more than one
# "operation" on a file.
TIMEOUT_PER_MB: float = 1.0
```

But it's not too critical. We can leave it as it.

Reply:
I agree, it's a bad explanation. Regarding `TIMEOUT_PER_MB = 1.0`, I think that people who are reading it will expect this to be a 1 second timeout per 1 MiB, but this is not the case here. How about `OPS_PER_MB`, since we already use this term? This fits nicely with this calculation (`ops = math.ceil(size / OPS_PER_MB)`).

Reply:
I have this feeling that I won't remember what op means in 6 months from now as well. Ok, let's make it simpler then:

```suggestion
# The timeout (in seconds) for a single page. If a command spans a number of
# pages, this number will be multiplied by the number of pages.
TIMEOUT_PER_PAGE: float = 10
# The timeout (in seconds) per MiB. If a command ingests files that are larger
# than 1 MiB, this number will be multiplied by their MiBs.
TIMEOUT_PER_MB: float = 1.0
# NOTE: For each operation on a file (be it a single page, multiple pages or a
# whole document) we calculate these two timeout values, and we use the
# largest one.
```

Is it clearer now?

Reply:
> Is it clearer now?

Way clearer. Thank you! So clear that I think we don't even need the long comment. I'd remove most of the comments since they are self-explanatory

```diff
-# The timeout (in seconds) for a single page. If a command spans a number of
-# pages, this number will be multiplied by the number of pages.
-TIMEOUT_PER_PAGE: float = 10
+TIMEOUT_PER_PAGE: float = 10  # (seconds)
-# The timeout (in seconds) per MiB. If a command ingests files that are larger
-# than 1 MiB, this number will be multiplied by their MiBs.
-TIMEOUT_PER_MB: float = 1.0
+TIMEOUT_PER_MB: float = 1.0  # (seconds)
-# NOTE: For each operation on a file (be it a single page, multiple pages or a
-# whole document) we calculate these two timeout values, and we use the
-# largest one.
```

The last note could possibly also go, since it's likely to be out of sync with the code if we ever change that (we won't remember there's a comment referencing it). Maybe moving it closer to the `max()`? (if the code isn't clear enough already).